src/site/styles/_our-community.scss

```diff
@@ -0,0 +1,37 @@
+@import "vars";
+
+.our-community {
+
+  .row {
+    padding: 5vh 0;
+
+    &:first-child {
+      padding-top: 0;
+    }
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $brand-dark;
+    }
+  }
+
+  .icon-container {
+    display: flex;
+
+    .icon {
+      width: 75%;
+      height: 100%;
+      fill: $brand-dark;
+    }
+  }
+
+  .page-hero {
+    background: url(/assets/our_community_hero_background.png), rgba(120,120,120,1);
```

Comment:
Please use the lighten function on the `$brand-dark`.

Reply:
ahhh, good call 👍